Changes needed to make the tailf tests work correctly.

These will be passed upstream.

--- util-linux-2.25.2/tests/ts/tailf/simple.orig	2015-02-16 18:13:47.493114932 -0800
+++ util-linux-2.25.2/tests/ts/tailf/simple	2015-03-01 15:18:29.040656789 -0800
@@ -27,9 +27,9 @@
 
 $TS_CMD_TAILF $INPUT > $TS_OUTPUT 2>&1 &
 
-sleep 0.5
+sleep 0.1
 echo {0..9} >> $INPUT
-sleep 0.5
+sleep 0.2
 
 rm -f $INPUT
 
